tests/websocket-clients/curl-client-yaml.sh

```diff
@@ -0,0 +1 @@
+curl -X POST http://localhost:8080/api/postKVP -H "Content-Type: application/octet-stream" --data-binary "@example.yaml"
```

Comment:
`curl -X POST -H "Content-Type: application/x-yaml" --data-binary @your_file.yaml <API_ENDPOINT>`

Should be x-yaml test also.